src/components/Calculator.js

```diff
@@ -1,70 +1,178 @@
 import React, { Component } from 'react';
+import calculate from '../logic/calculate';
 
 // eslint-disable-next-line react/prefer-stateless-function
 class Calculator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+    this.myHandleClick = this.myHandleClick.bind(this);
+  }
+
+  myHandleClick(event) {
+    this.setState((state) => calculate(state, event.target.textContent));
+  }
+
   render() {
+    const { next, total } = this.state;
     return (
       <div className="flex justify-center items-center h-screen">
         <div className="w-2/6 shadow-lg">
           <div className="text-white bg-gray-500/90 flex justify-end p-5 w-full">
-            <span>0</span>
+            <span>{next || total || 0}</span>
           </div>
           <table className="w-full">
             <tbody className="w-2/6">
               <tr>
```

Comment:
[OPTIONAL]

- Kindly ensure that your UI design looks like the design template given to you for this project so that your app can look very nice 😄 

Your Design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/188995575-1a0c3079-11a3-4275-8a3e-2b1de6f8094d.png)

Figma Design 👇🏽 

![image](https://user-images.githubusercontent.com/63473584/188995714-7ff5b250-73b0-4187-b330-bc42af0406d3.png)